compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/calls/FirArgumentsToParametersMapper.kt

```diff
@@ -74,6 +78,7 @@ fun mapArguments(
                 name = function.valueParameters.last().name
             }
             argumentsInParenthesis = argumentsInParenthesis.dropLast(1) + listOf(namedV)
+            oldToNewArgumentMap[v] = namedV
```

Comment:
This is used to replace the set argument in the call's argument list (see changes in `FirCallCompletionResultsWriterTransformer.kt`). Otherwise `remapArgumentsWithVararg` in `BodyResolveUtils.kt` simply loses the argument (it remaps based on the call argument list). I'm open to other suggestions on how to solve this.